packages/orbit-components/src/primitives/TooltipPrimitive/index.tsx

```diff
@@ -24,7 +24,7 @@ export const TooltipWrapper = React.forwardRef<
         block ? "flex" : "inline-flex",
         enabled &&
           !removeUnderlinedText &&
-          "[&_.orbit-text]:inline-block [&_.orbit-text]:underline [&_.orbit-text]:decoration-current [&_.orbit-text]:decoration-dotted",
+          "[&_.orbit-text]:orbit-decoration-current [&_.orbit-text]:inline-block [&_.orbit-text]:underline [&_.orbit-text]:decoration-dotted",
```

Comment:
For the same reason as the `bg-inherit` that I reported on Slack, this also needs to be accepted in our config so that we accept currentColor. So the class needs to be as it was. Let me know if you need further assistance with adding inherit and currentColor as valid values for the colors in our config